resources/aws_ec2.go

```diff
@@ -708,3 +729,21 @@ func (obj *AwsEc2Res) snsPostHandler(w http.ResponseWriter, req *http.Request) {
 		log.Printf("%s: Post: %s", obj, string(post))
 	}
 }
+
+// snsMakeTopic creates a topic on aws sns.
+func (obj *AwsEc2Res) snsMakeTopic() error {
```

Comment:
any reason why the function signature isn't something like (string, error) where the string value is the topicArn I suppose? In particular, while looking at pr#242 I don't actually see this function being called anywhere. Should it happen in init? in checkapply?